### ISSUE TITLE
ECANCELED error frequently logged to console.error when calling channel.stop

### ISSUE THREAD

**Issue by bbrowning** [projectodd/aquila#3]
I quite often see multiple instances of the below error logged when calling channel.stop

```
{ [Error: send ECANCELED] code: 'ECANCELED', errno: 'ECANCELED', syscall: 'send' }
```

It's coming from https://github.com/projectodd/aquila/blob/0e347ac99f198d6c9c57a5ca5780586673540c4a/lib/protocol/udp.js#L58, which may be unnecessary since we also call deferred.reject with the error.

**Comment by lance** [comment]
@bbrowning I think you are probably right - may really be that we don't need console.error here, since we inform the caller with deferred.reject - informing the user via console or whatever is probably more the client code's responsiblity

**Comment by lance** [comment]
Addressed in 584747bf79aa9a28cc978fbe79416b52c2e37fc4

**Comment by bbrowning** [comment]
Thanks!
